runtime/src/proofs.rs

```diff
@@ -52,6 +49,9 @@ pub fn validate_proofs(doc_root: H256, proofs: &Vec<Proof>, static_proofs: [H256
         .fold(true, |acc, b| acc && b);
 }
 
+// computes blake2 256 sorted hash of the a and b
+// if a < b: blake256(a+b)
+// else: blake256(b+a)
```

Comment:
@vedhavyas do we still need this? Have not really been involved in this, but I remember vaguely that hashes are now always sorted?

Reply:
yeah, we still need this. only the signing_root, signature_root, and the two data roots are sorted. Rest all profs which lead up to these above rots are calculated through sorted hashes

Reply:
An Example of this would be proof for a field in Invoice document. The proofs that come with this are to be sorted to lead up to Data rot